### PR TITLE
Removes videoId from URL

### PR DESCRIPTION
**Pull request by jaysaleh**
This pr removes the videoId from the URL

### CODE REVIEW

File: src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java [hunk 1]
```diff
@@ -94,7 +94,7 @@ public void doPost_urlAlreadyInDatabase_shouldReturnLecture() throws IOException
     servlet.doPost(request, response);
 
     assertThat(datastoreService.prepare(new Query(LectureUtil.KIND)).countEntities()).isEqualTo(1);
-    verify(response).sendRedirect("/view/?id=1&video-id=wXhTHyIgQ_U");
+    verify(response).sendRedirect("/view/?id=1");
```
Comment (amuamushu) [inline src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java:97]:
```suggestion
    verify(response).sendRedirect("/view?id=1");
```
I don't think there should be an extra slash after `view`. I think the issue may be the `REDIRECT_URL` in `LectureServlet`.

File: src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java [hunk 2]
```diff
@@ -105,7 +105,7 @@ public void doPost_urlNotInDatabase_shouldAddToDatabaseAndReturnRedirect() throw
...
```
Comment (amuamushu) [inline src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java:108]: here too, extra slash
```suggestion
    verify(response).sendRedirect("/view?id=1");
```

File: src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java (hunk 1 above)
Comment (jaysaleh) [reply src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java:97]: fixing in a separate pr

File: src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java [hunk 3]
```diff
@@ -152,17 +152,14 @@ private Entity getLectureEntityFromRequest(HttpServletRequest request) {
   }
 
   /**
-   * Returns URL redirecting to lecture view page with parameters {@code lectureId}
-   * and {@code videoId} found in {@code lectureEntity}.
+   * Returns URL redirecting to lecture view page with parameter {@code lectureId}
```
Comment (n-wach) [inline src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java:155]: Please explain `lectureId` in the comment, or remove.

I think: `Returns URL for the lecture view page for {@code lectureEntity}` might be good enough

Comment (jaysaleh) [reply src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java:155]: Fixed

File: src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java (hunk 1 above)
Comment (n-wach) [reply src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java:97]: Technically I think it should work with or without the slash, but yeah we should be consistent and not use one.

**Comment by jaysaleh** [comment]
> Does the transcript stuff work correctly without a video id in the URL? Or is it used somewhere. I notice the window.location.search is still passed as an arg in loadTranscript

Right now the transcripts don't work, was thinking we could do it in a separate PR

File: src/main/webapp/view/transcript/transcript.js [hunk 4]
```diff
@@ -21,6 +21,7 @@ const ENDPOINT_TRANSCRIPT = '/transcript';
  */
 // TODO: Delete this method once TranscriptServlet's doPost()
 // is called in LectureServlet.
+// TODO: Add videoId as a param.
```
Comment (billie-cat) [inline src/main/webapp/view/transcript/transcript.js:24]: Why add this if the other TODO says to delete the method?

Comment (jaysaleh) [reply src/main/webapp/view/transcript/transcript.js:24]: For transcripts to work while this function exists the param needs to be added

**Pull request merged**